data/presets/highway/ladder.json

```
@@ -0,0 +1,31 @@
+{
+    "icon": "temaki-crossing_markings-ladder",
+    "fields": [
+        "name",
+        "incline_steps",
+        "access_simple",
+        "handrail",
+        "step_count",
+        "surface",
```

I'd replace `surface` with `material`, as `surface` does not make much sense conceptually for ladders, and the offered values would be silly (e.g. `unpaved`, `sand`, `ground`, `asphalt`, etc. are pretty unlikely to fit for a ladder :stuck_out_tongue_winking_eye: ).

```suggestion
        "material",
```